ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ManagementResources.scala

```diff
@@ -203,6 +206,17 @@ class ManagementResources(processCounter: ProcessCounter,
             }
           }
         }
+      } ~
+      path("processManagement" / "customAction" / Segment) { processName =>
+        (post & processId(processName) & entity(as[CustomActionRequest])) { (process, req) =>
+          complete {
+            (managementActor ? (req.toEngineRequest(process.name), process.id, user))
```

Comment:
Maybe let's add case class for this tuple to be more consistent with other actions?

Reply:
:+1: